interface/main/left_nav.php

```diff
@@ -1358,6 +1358,7 @@ function selpopup(selobj) {
       </li>
       <li><a class="collapsed_lv2"><span><?php xl('Visits','e') ?></span></a>
         <ul>
+          <?php  genMiscLink('RTop','rep','0',xl('Daily Summary Report'),'reports/daily_summary_report.php'); ?>
```

Comment:
A shorter name will avoid formatting issues - either browser adding horizontal scroll or text getting truncated if overflow is suppressed or multi-line menu item.